src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -253,10 +253,16 @@ public UploadResponse uploadDocuments(
     public ResponseEntity<PatchDocumentResponse> patchDocumentByDocumentId(
         @ApiParam(value = "", required = true)
         @Valid @RequestBody final UpdateDocumentCommand body,
+
+        final BindingResult bindingResult,
```

Comment:
i think we don't need BindingResult in all these controller methods except "Multi form data" one